corehq/apps/cloudcare/templates/cloudcare/bootstrap5/preview_app_base.html

```diff
@@ -57,6 +57,7 @@
 
   {# This is fine as an inline script; it'll be removed once form designer is migrated to RequireJS #}
   <script>
+    window.USE_BOOTSTRAP5 = true;
```

Comment:
will that stay in? or is that just for testing the PR?

Reply:
It'll stay for the duration of the web apps migration. Once we're switched over to B5 templates, I'll do a cleanup PR that'll remove the B3 templates and also references to `USE_BOOTSTRAP5`, including this line.